etc/config/c++.amazon.properties

```diff
@@ -1446,6 +1446,21 @@ libs.immer.versions=trunk
 libs.immer.versions.trunk.version=trunk
 libs.immer.versions.trunk.path=/opt/compiler-explorer/libs/immer/trunk
 
+libs=nsimd
+libs.nsimd.name=NSIMD
+libs.nsimd.url=https://github.com/agenium-scale/nsimd/
+libs.nsimd.versions=x86_64:arm:arm64
```

Comment:
These versions don't feel correct to me - what do we expect to happen once the next version of nsimd is released?
Typically we keep around all tagged versions of libraries and add newer versions.

I think we have to do something like `libs.nsimd.versions.x86_64-22`

Reply:
@apmorton ok to do that, do we need to do something for you concerning this?